gocourse08/main.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
```

Comment:
```suggestion
	"math/rand/v2"
```

Ось тут почитати, чому так https://go.dev/blog/randv2